### PR TITLE
added new template

### PR DESCRIPTION
**Pull request by maikvandergaag**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 101-webapp-including-insights/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "webappName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The name of the web application"
+      }
+    },
+    "appInsightsName": {
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.json]: move to variable

File: 101-webapp-including-insights/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The name of the application insights instance"
+      }
+    },
+    "hostingPlanName": {
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.json]: move to variable

File: 101-webapp-including-insights/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The name of the hosting plan for the web application"
+      }
+    },
+    "skuName": {
+      "type": "string",
+      "defaultValue": "F1",
+      "allowedValues": [
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.json]: remove allowedValues

File: 101-webapp-including-insights/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+      "metadata": {
+        "description": "The hosting plan's instance count"
+      }
+    }
+  },
+  "variables": {
+  },
+  "resources": [
+    {
+      "apiVersion": "2016-09-01",
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.json]: update apiVersion

File: 101-webapp-including-insights/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+      "sku": {
+        "name": "[parameters('skuName')]",
+        "capacity": "[parameters('skuCapacity')]"
+      },
+      "properties": {
+        "name": "[parameters('hostingPlanName')]"
+      }
+    },
+    {
+      "apiVersion": "2015-08-01",
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.json]: update apiVersion

File: 101-webapp-including-insights/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+        "[resourceId('Microsoft.Web/serverfarms/', parameters('hostingPlanName'))]",
+        "[resourceId('microsoft.insights/components/', parameters('appInsightsName'))]"
+      ],
+      "properties": {
+        "name": "[parameters('webappName')]",
+        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', parameters('hostingPlanName'))]"
+      },
+      "resources": [
+        {
+          "apiVersion": "2015-08-01",
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.json]: update apiVersion

File: 101-webapp-including-insights/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/Sites', parameters('webappName'))]",
+            "Microsoft.ApplicationInsights.AzureWebSites"
+          ],
+          "properties": {
+            "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(concat('microsoft.insights/components/', parameters('appInsightsName'))).InstrumentationKey]"
+          }
+        },
+        {
+          "apiVersion": "2015-08-01",
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.json]: update apiVersion

File: 101-webapp-including-insights/azuredeploy.parameters.json [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    "webappName": {
+      "value": "GEN-UNIQUE"
+    },
+    "appInsightsName": {
+      "value": "GEN-UNIQUE"
+    },
+    "hostingPlanName": {
+      "value": "GEN-UNIQUE"
+    },
+    "skuName": {
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.parameters.json]: remove this and use a defaultValue in the template

File: 101-webapp-including-insights/azuredeploy.parameters.json [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+      "value": "S1"
+    },
+    "owner": {
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.parameters.json]: remove this and use a defaultValue in the template

File: 101-webapp-including-insights/azuredeploy.parameters.json [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+      "value": "owner"
+    },
+    "skuCapacity": {
```
Comment (bmoore-msft) [inline 101-webapp-including-insights/azuredeploy.parameters.json]: remove this and use a defaultValue in the template

File: 101-webapp-including-insights/azuredeploy.json (hunk 3 above)
Comment (maikvandergaag) [reply 101-webapp-including-insights/azuredeploy.json]: done

File: 101-webapp-including-insights/azuredeploy.json (hunk 4 above)
Comment (maikvandergaag) [reply 101-webapp-including-insights/azuredeploy.json]: done

File: 101-webapp-including-insights/azuredeploy.json (hunk 5 above)
Comment (maikvandergaag) [reply 101-webapp-including-insights/azuredeploy.json]: which version should this be. The config element is not present in the documentation

File: 101-webapp-including-insights/azuredeploy.parameters.json (hunk 8 above)
Comment (maikvandergaag) [reply 101-webapp-including-insights/azuredeploy.parameters.json]: done

File: 101-webapp-including-insights/azuredeploy.json (hunk 7 above)
Comment (maikvandergaag) [reply 101-webapp-including-insights/azuredeploy.json]: done

File: 101-webapp-including-insights/azuredeploy.parameters.json (hunk 10 above)
Comment (maikvandergaag) [reply 101-webapp-including-insights/azuredeploy.parameters.json]: done

File: 101-webapp-including-insights/azuredeploy.parameters.json (hunk 9 above)
Comment (maikvandergaag) [reply 101-webapp-including-insights/azuredeploy.parameters.json]: done

**Pull request closed without merging**